### ISSUE TITLE
fails when box is in FIPS mode

### ISSUE THREAD

**Issue by davidkarlsen** [anchore/anchore-engine#882]
```
anchore-engine-anchore-engine-catalog-7b9b766788-sbcjg anchore-engine-catalog [service:catalog] 2021-01-27 17:19:27+0000 [-] [Thread-729] [anchore_engine.clients.services.internal/dispatch()] [ERROR] Failed client call to service simplequeue for url: http://anchore-engine-anchore-engine-simplequeue:8083/v1/queues/watcher_tasks/is_inqueue. Response: {'httpcode': 500, 'anchore_error_raw': 'b\'"[digital envelope routines: EVP_DigestInit_ex] disabled for FIPS"\\n\'', 'anchore_error_json': '[digital envelope routines: EVP_DigestInit_ex] disabled for FIPS'}
```

See similar issue and how to avoid it: https://github.com/s3tools/s3cmd/issues/1005

chart version: 

```
helm list
NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
anchore-engine  anchore         1               2021-01-27 17:07:11.379189889 +0000 UTC deployed        anchore-engine-1.11.5   0.9.0   
```

**Comment by zhill** [comment]
thanks @davidkarlsen , the pointer to a solution is appreciated. we just ran into this ourselves as well so we'll update here.

**Comment by davidkarlsen** [comment]
@zhill any idea which release it will go into?

**Comment by davidkarlsen** [comment]
bump

**Comment by davidkarlsen** [comment]
https://github.com/psycopg/psycopg2/issues/1170

**Comment by zhill** [comment]
Hi @davidkarlsen I've got some changes in testing, in #985. If things look ok when we do more testing, then either 0.9.4 or the next minor release will have the fix.

**Comment by zhill** [comment]
Have been testing the #985 fix and more updates are needed to handle SQLAlchemy (may need to monkey patch it). Will update here, but did not make 0.9.4. We are going to have to get FIPS compliant test infrastructure to ensure this is correct in CI, so may take more time than available for the next feature release as well. But, we are working on it and will continue updates as we make progress.

**Comment by ryphon** [comment]
Hey @zhill, just ran into basically this exact same issue on a RHEL7 installation. Any chance there's any updates on this since mid May?

**Comment by zhill** [comment]
Closed by #1193 and released in v0.10.2
